### ISSUE TITLE
Language is lowercased, as opposed to the example

### ISSUE THREAD

**Issue by GrosSacASac** [hapijs/accept#61]
#### Support plan

* *is this issue currently blocking your project?* (no):
* *is this issue affecting a production system?* (/no):

#### Context

* *node version*: 15.2.1
* *module version with issue*: @hapi/accept 5.0.1
* *last module version without issue*:  ?
* *environment* (e.g. node, browser, native):  node
* *used with* (e.g. hapi application, another framework, standalone, ...): standalone or http
* *any other relevant information*:

#### What are you trying to achieve or the steps to reproduce?

I try to get the preferred language from a http request

```
node
Welcome to Node.js v15.2.1.
Type ".help" for more information.
```

```js
> Accept=require("@hapi/accept")

const language = Accept.language("en;q=0.7, en-GB;q=0.8");
> language
'en-gb'
```

#### What was the result you got?

```js
'en-gb'
```
#### What result did you expect?

Same as in the example found here https://hapi.dev/module/accept/api/?v=5.0.1

```js
const language = Accept.language("en;q=0.7, en-GB;q=0.8"); // language === "en-GB"
'en-GB'
```

**Comment by Nargonath** [comment]
This seems to be the case for at least 2 years according to `git blame` on this test: https://github.com/hapijs/accept/blob/master/test/language.js#L26. I believe that's just a mismatch between the documentation and the feature. Feel free to open a PR to fix the documentation.

**Comment by GrosSacASac** [comment]
Actually I would prefer if the code was changed. And the reason is that the it allows to send the correct [Content-Language HTTP header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Language) back in thee response.

**Comment by kanongil** [comment]
Why? The `Content-Language` language-tags are supposed to be [case insensitive](https://tools.ietf.org/html/rfc7231#section-3.1.3.1).

If you care about how it looks, you can reformat it according to the rules here: https://tools.ietf.org/html/rfc5646#section-2.1.1

**Comment by GrosSacASac** [comment]
You are right, casing does not have meaning. However casing is recommended to aid users if I understand correctly.

So, final decision, update doc or code ?

**Comment by Nargonath** [comment]
@GrosSacASac I'd lean towards updating the docs.

**Comment by GrosSacASac** [comment]
Where is the source of the docs ?

**Comment by kanongil** [comment]
https://github.com/hapijs/accept/blob/master/API.md

**Comment by GrosSacASac** [comment]
https://github.com/hapijs/accept/pull/62

**Comment by devinivy** [comment]
Closed by #62, thanks all!
